modules/storage-persistent-storage-pvc.adoc

```diff
@@ -113,3 +113,63 @@ spec:
 <1> Path to mount the volume inside the pod.
 <2> Name of the volume to mount. Do not mount to the container root, `/`, or any path that is the same in the host and the container. This can corrupt your host system if the container is sufficiently privileged, such as the host `/dev/pts` files. It is safe to mount the host by using `/host`.
 <3> Name of the PVC, that exists in the same namespace, to use.
+
+[id="pvc-cli-command-usage_{context}"]
+== Viewing PVC usage statistics
+
+You can view usage statistics for persistent volume claims (PVCs).
```

Comment:
I assume the target audience likely knows why they want to view usage statistics for PVCs, but if we have more information about why you'd want to complete this task, it would be good to add it here.